perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/MembersManagerBlImpl.java

```diff
@@ -3729,4 +3740,30 @@ private String getUserLogin(PerunSession sess, User user, String namespace) {
 			throw new InternalErrorException("Failed to get namespace login for user: " + user, e);
 		}
 	}
+
+	/**
+	 * Resolve email template (subject/message) from an entityless attribute.
+	 * @param sess Perun session
+	 * @param attributeName friendly name of the attribute (namespace of entityless is assigned in the method)
+	 * @param language Language of the template. If fails to find, default EN template will be looked up.
+	 * @return Found template for given language. If fails, tried to find for default EN. If fails, returns null.
+	 * @throws AttributeNotExistsException
+	 */
+	private String getEmailTemplateFromEntitylessAttribute(PerunSession sess, String attributeName, String language)
+		throws AttributeNotExistsException
+	{
+		String template;
+		try {
+			attributeName = AttributesManager.NS_ENTITYLESS_ATTR_DEF + ':' + attributeName;
+			Attribute templateAttribute = perunBl.getAttributesManagerBl().getAttribute(sess, language, attributeName);
+			template = (String) templateAttribute.getValue();
+			if (template == null) {
+				templateAttribute = perunBl.getAttributesManagerBl().getAttribute(sess, "en", attributeName);
+				template = (String) templateAttribute.getValue();
+			}
```

Comment:
Attributes have helper methods to get value with correct type.
```suggestion
			template = templateAttribute.valueAsString();
			if (template == null) {
				templateAttribute = perunBl.getAttributesManagerBl().getAttribute(sess, "en", attributeName);
				template = templateAttribute.valueAsString();
			}
```